Gordon360/Services/ProfileService.cs

```diff
@@ -63,7 +65,56 @@ public AlumniProfileViewModel GetAlumniProfileByUsername(string username)
                 result = alumni;
             return result;
         }
+        /// <summary>
+        /// get advisors for particular student
+        /// </summary>
+        /// <param name="id">student id</param>
+        /// <returns></returns>
+        public IEnumerable<AdvisorViewModel> GetAdvisors(string id)
+        {
+            //var query = RawSqlQuery<ADVISOR_SEPARATE_Result>.query("ADVISOR_SEPARATE @ID", new SqlParameter("ID", SqlDbType.VarChar) { Value = id });
+            var query = _unitOfWork.AccountRepository.FirstOrDefault(x => x.gordon_id == id);
```

Comment:
What does this line do?

Reply:
Looking at GetPhotoPath(), which this line is copied from, it appears this is a check that the ID is valid (that is, represents a real user).